src/transport/receiver.ts

```diff
@@ -36,64 +14,13 @@ export class Receiver {
    * @param {Object|JSON} body The body of the HTTP request
    * @return {CloudEvent} A new {CloudEvent} instance
    */
-  static accept(
-    headers: Headers,
-    body: string | Record<string, unknown> | CloudEventV1 | CloudEventV03 | undefined | null,
-  ): CloudEvent {
+  accept(headers: Headers, body: string | Record<string, unknown> | undefined | null): CloudEvent {
```

Comment:
Should we have a deprecation message printout when this is called?

Is it also possible in Typescript to deprecate a function using the jsdoc annotations?

Reply:
Yes - adding `@deprecated` results in doc output that looks like this:

![image](https://user-images.githubusercontent.com/15952/91210776-bb21c400-e6db-11ea-9644-1bb00bc84be8.png)

Reply:
Nice,  probably want add the same to the emitter?